prov/efa/src/efa_hmem.c

```diff
@@ -32,21 +32,62 @@
 
 #include "efa.h"
 
+#if HAVE_CUDA || HAVE_NEURON
+static size_t efa_max_eager_msg_size_with_largest_header(struct efa_domain *efa_domain) {
+	int mtu_size;
+
+	mtu_size = efa_domain->device->rdm_info->ep_attr->max_msg_size;
+	if (rxr_env.mtu_size > 0 && rxr_env.mtu_size < mtu_size)
+		mtu_size = rxr_env.mtu_size;
+	if (mtu_size > RXR_MTU_MAX_LIMIT)
+		mtu_size = RXR_MTU_MAX_LIMIT;
+
+	return mtu_size - rxr_pkt_max_hdr_size();
+}
+#endif
+
+/**
+ * @brief Update the System hmem_info struct
+ *
+ * @param system_info[out]	System hmem_info struct
+ * @return 0 on success
+ */
+static int efa_hmem_info_update_system(struct efa_hmem_info *system_info)
+{
+	system_info->initialized = true;
+	system_info->p2p_supported = true;
+	system_info->max_medium_msg_size = rxr_env.efa_max_medium_msg_size;
+
+	/* Default documented in rxr_env.c */
```

Comment:
unnecessary comment

Reply:
got rid of all default documented comments... The intention of them was to notify the developer that they shouldn't change this value here without updating the docs in rxr_env.c

Reply:
> The intention of them was to notify the developer that they shouldn't change this value here without updating the docs in rxr_env.c

We should abstract such constant values behind macros. I know these hardcoded constants were in place before your PR, so it can be a separate task (for you or anyone).